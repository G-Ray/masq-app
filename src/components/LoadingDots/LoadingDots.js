import React from 'react'

import './LoadingDots.css'

export default function LoadingDots () {
  return (
    <div className='Loading'>
      <div className='loading-dots' />
      <div className='loading-dots' />
      <div className='loading-dots' />
    </div>
  )
}
