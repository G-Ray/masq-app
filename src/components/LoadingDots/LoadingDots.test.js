import React from 'react'
import renderer from 'react-test-renderer'
import LoadingDots from './LoadingDots'

it('renders LoadingDots', () => {
  const component = renderer.create(<LoadingDots />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
