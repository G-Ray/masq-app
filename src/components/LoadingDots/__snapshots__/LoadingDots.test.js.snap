// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LoadingDots 1`] = `
<div
  className="Loading"
>
  <div
    className="loading-dots"
  />
  <div
    className="loading-dots"
  />
  <div
    className="loading-dots"
  />
</div>
`;
